azurerm/internal/services/web/tests/resource_arm_app_service_test.go

```diff
@@ -3981,6 +4024,44 @@ resource "azurerm_app_service" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger)
 }
 
+func testAccAzureRMAppService_withSourceControl(data acceptance.TestData, branch string) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
we should include the service here
```suggestion
  name     = "acctestRG-web-%d"
```